.travis.yml

```diff
@@ -0,0 +1,83 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  global:
+    - MBEDTLS_VERSION=2.2.1
+    - MBEDTLS_PREFIX=~/opt/mbed
+    - MBEDTLS_LIBS="-L$MBEDTLS_PREFIX/lib -lmbedtls -lmbedx509 -lmbedcrypto"
+    - MBEDTLS_CFLAGS="-I$MBEDTLS_PREFIX/include"
+
+matrix:
+  include:
+    - env: CONFIG="--disable-lzo"
```

Comment:
Why --disable-lzo?  I see you've installed liblzo2-dev, that should be sufficient, right?